bitcrowd/bitstyles
# Update Section Heading component

Fixes #783 


## 📸 Looks like

![image](https://github.com/bitcrowd/bitstyles/assets/61787049/ca3b96da-ddad-47b3-b28a-975942fd3cd2)

#### Small screens
<img width="417" alt="image" src="https://github.com/bitcrowd/bitstyles/assets/61787049/a4cc9017-26c5-4251-985d-794999b0bedf">

#### Large screens
<img width="1398" alt="image" src="https://github.com/bitcrowd/bitstyles/assets/61787049/9be24e2d-cd9d-4774-b966-76a19e270ec3">


#### Changes
- [x] On smaller screens items stack top to bottom from the left to right.
- [x] When an item is attached to the title text (i.e. a badge or counter) it sticks/aligns to the top line of text when the text breaks to multiple lines.
- [x] All Section Headings have a 2px border underline.
- [x] The border underline fills the container width (minus padding)

## How to QA

- `git fetch`
- `git checkout <pr-branch-name>`
- `yarn`
- `yarn storybook`
- navigate to `Section heading`

Check:

- [x] The appearance matches the design in Zeplin
- [x] The Zeplin component is correctly linked in storybook

### 👾 Code changes

Check:
